releasenotes/notes/add-changelog-4f9596bd058cca9c.yaml

```diff
@@ -0,0 +1,15 @@
+---
+prelude: >
+    Significantly simplify the penaltymodel ecosystem. ``penaltymodel`` is now a single
+    package with a simplified interface.
+features:
```

Comment:
Don't these (all under `prelude` line usually have a 

```
- |
```

line?

Reply:
That's just for multiline comments or if there is a leading `` ` ``